### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by alexkocharyan01**

### CODE REVIEW

**Review by ArthurO2002 — APPROVED** [review]

File: src/validation/support/contactUs.ts [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+      /^[aA-zZ\s]+$/,
+      i18next.t(Translation.PAGE_CONTACT_US_VALIDATION_NAME_MATCHES) as string
+    )
+    .required(i18next.t(Translation.PAGE_CONTACT_US_VALIDATION_NAME) as string),
   email: string()
     .email()
-    .matches(/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/, 'Invalid Email address')
-    .required('Email is required'),
+    .matches(
+      /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/,
```
Comment (ArthurO2002) [inline src/validation/support/contactUs.ts:15]: Let's move to constants, not sure why we are validating mail with regex

File: src/validation/support/contactUs.ts [hunk 2]
```diff
@@ -1,18 +1,26 @@
+import { Translation } from '@constants/translations'
+import i18next from 'i18next'
 import { object, string } from 'yup'
 
 export const contactUsValidationSchema = object({
   fullName: string()
-    .matches(/^[aA-zZ\s]+$/, 'Only alphabets are allowed for this field')
-    .required('Full name is required'),
+    .matches(
+      /^[aA-zZ\s]+$/,
```
Comment (ArthurO2002) [inline src/validation/support/contactUs.ts:8]: Move to constants

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/validation/support/contactUs.ts (hunk 1 above)
Comment (alexkocharyan01) [reply src/validation/support/contactUs.ts]: done

File: src/validation/support/contactUs.ts (hunk 2 above)
Comment (alexkocharyan01) [reply src/validation/support/contactUs.ts]: done

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

**Review by Arthur02V — APPROVED** [review]

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**